app/src/main/java/edu/wpi/first/shuffleboard/app/WidgetPaneController.java

```diff
@@ -124,6 +124,23 @@ private void initialize() {
       event.consume();
     });
 
+    // Drag tiles outside the pane to remove them
+    pane.setOnDragExited(e -> {
```

Comment:
I'm worried that this will remove a widget as soon as we drag it away, instead of removing a widget when we *release* the drag.

Reply:
The widget pane DnD handler isn't able to know when the drag gesture finishes if it's not on the widget pane. It only knows when the drag exits, enters, or is released _on the widget pane_.